seleniumDescriptors.js

```diff
@@ -0,0 +1,177 @@
+var format = require('util').format;
```

Comment:
Single responsibility principle. I wanted SeleniumTunnel to have a single responsibility. seleniumDescriptors has multiple configurations, but they also adhere to (mostly) a single theme. Combining the two added more complexity to the module than I wanted.